app/assets/stylesheets/bookshow.css

```diff
@@ -0,0 +1,160 @@
+.showbook-section {
+  padding-top: 20px;
+  border-top: 1px #bfbfbf solid;
```

Comment:
Since you are using this many times you could create a variable, `border_style` for example.
For this you will have to add the `.scss` extension to this file.